src/components/DevicesView.jsx

```diff
@@ -34,16 +34,24 @@ const DevicesView = ({ t, f, isFetching, devices, openDeviceRevokeModale, device
 
         <div className={classNames(tableStyles['coz-table-head'], tableStyles['coz-table-row'])}>
           <div className={classNames(tableStyles['coz-table-header'], tableStyles['set-table-name'])}>{ t('DevicesView.head_name') }</div>
+          <div className={classNames(tableStyles['coz-table-header'], tableStyles['set-table-date'])}>{ t('DevicesView.head_sync') }</div>
```

Comment:
Is this syntax documented somewhere ? 
`tableStyles['coz-table-header'], tableStyles['set-table-date']`

That's... strange, if you make a table, you should use the <table> tag, this syntax isn't semantic, and not accessible.